src/intree.js

```diff
@@ -23,10 +24,6 @@ function completeInTreeConfig(config, payload) {
     ];
   }
 
-  config.routes = [
-    `taskcluster-github.${ payload.organization }.${ payload.repository }.` + payload.details['event.head.sha'],
```

Comment:
We could add it to every task I suppose, but I don't think anyone has noticed it missing.